de.metas.adempiere.adempiere/base/src/main/java/de/metas/translation/api/impl/ElementTranslationBL.java

```diff
@@ -62,8 +67,44 @@ public void updateFieldTranslationsFromAD_Name(int elementId)
 
 	}
 
+	@Override
+	public void updateWindowTranslationsFromElement(int elementId)
+	{
+		final String trxName = ITrx.TRXNAME_ThreadInherited;
+		{
```

Comment:
why does extra `{` `'}'?

same for the methods below...

Reply:
Fixed